docs/operation/preparation.rst

```diff
@@ -8,5 +8,5 @@ with the current installation:
 
    .. code::
 
-     /srv/scality/metalk8s-X.X.X/iso-manager.sh -p <path_to_iso>
+     /srv/scality/metalk8s-X.X.X/iso-manager.sh -a <path_to_iso>
```

Comment:
Could we use the long option name instead? I always find it more obvious when used in scripts (let's keep the short name for manual interactions)